Program.cs

```diff
@@ -1,18 +1,43 @@
 ﻿using MongoDB.Driver;
 using MongoDB.Bson;
+using System.Diagnostics;
 
 public static class Example
 {
     public static void Main(string[] args)
     {
-        var client = new MongoClient(args[0].ToString());
+        var connectionUri = args[0].ToString();
+        var settings = MongoClientSettings.FromConnectionString(connectionUri);
+        var client = new MongoClient(settings);
 
         IMongoDatabase db = client.GetDatabase("test");
         var command = new BsonDocument { { "ping", 1 } };
         var res = db.RunCommand<BsonDocument>(command);
+        Console.WriteLine(res);
 
+        command = new BsonDocument { { "dropDatabase", 1 } };
+        res = db.RunCommand<BsonDocument>(command);
         Console.WriteLine(res);
 
+        var documentList = new List<BsonDocument>();
+        for (int i = 1; i < 5; i++)

```

Comment:
Isn't it idiomatic to start loop from i = 0?

Reply:
Actually it only has 4 items.

For an example, I think the readability improves to have it without for loop like below. This is just my preference :hugs:  FYI not tested code so would not compile.
```
 var documentList = new List<BsonDocument>{
new BsonDocument{ {"_id", 0 }, { "a", 0 } },
new BsonDocument{ {"_id", 1 }, { "a", 1 } },
new BsonDocument{ {"_id", 2 }, { "a", 2 } },
new BsonDocument{ {"_id", 3 }, { "a", 3 } },
}
```